### DISCUSSION TITLE
Better Chat Plus emotes

### DISCUSSION THREAD

**Discussion by bean-frog** [AntiCope/orion#30] [General]
is there a list anywhere of what emotes are supported in the better chat plus module?

**Comment by C10udburst** [comment]
`:smile:` -> ☺
`:sad:` -> ☹
`:heart:` -> ❤
`:skull:` -> ☠
`:star:` -> ★
`:flower:` -> ❀
`:pick:` -> ⛏
`:wheelchair:` -> ♿
`:lightning:` -> ⚡
`:rod:` -> 🎣
`:potion:` -> 🧪
`:fire:` -> 🔥
`:shears:` -> ✂
`:bell:` -> 🔔
`:bow:` -> 🏹
`:trident:` -> 🔱
`:cloud:` -> ☁

**Reply by bean-frog** [reply]
thank you!
